timescaledb/overview/release-notes/index.md

```diff
@@ -15,11 +15,8 @@ we welcome you to visit our Github repo or join our [Slack community](https://sl
 
 For our next release, we plan to add:
 
-- Elasticity: Move data when expanding or shrinking a multi-node cluster
-- Experimental schema: A way to get experimental functionalities early to our users
-- Various bug fixes.
-
-The current generally available (GA) version is 2.3.1.
+- Continuous aggregates for distributed hypertables
```

Comment:
Are we committing to adding both features for the next release?

Reply:
These are current priorities. :)